config/routes.rb

```diff
@@ -42,6 +42,9 @@
     end
   end
 
+  resources :link_report do
```

Comment:
I think this should just be a single POST (or PATCH) route, rather than using `resources` and creating seven routes. You can see all of the routes that created by running `rake routes`.